boards/opentitan/earlgrey-cw310/layout.ld

```diff
@@ -12,8 +12,28 @@ MPU_MIN_ALIGN = 1K;
 SECTIONS {
     .manifest ORIGIN(rom):
     {
-      . = . + 892;
-      LONG(_stext)
+        /* sw/device/silicon_creator/lib/manifest.h */
+        . = . + 384; /* signature */
+        . = . + 4;   /* usage_constraints.selector_bits */
+        . = . + 32;  /* usage_constraints.device_id */
+        . = . + 4;   /* usage_constraints.manuf_state_creator */
+        . = . + 4;   /* usage_constraints.manuf_state_owner */
+        . = . + 4;   /* usage_constraints.life_cycle_state */
+        . = . + 384; /* modulus */
+        . = . + 4;   /* address_translation */
+        . = . + 4;   /* identifier */
+        . = . + 4;   /* length */
+        . = . + 4;   /* version_major */
+        . = . + 4;   /* version_minor */
+        . = . + 4;   /* security_version */
+        . = . + 8;   /* timestamp */
+        . = . + 32;  /* binding_value */
+        . = . + 4;   /* max_key_version */
+        . = . + 4;   /* code_start */
+        . = . + 4;   /* code_end */
+        LONG(_stext - ORIGIN(rom)); /* . = . + 4; entry_point */
+        . = . + 128; /* padding */
+        /* size = 1024 bytes */
```

Comment:
Thanks for this.

This doesn't work for me though. I had to change `LONG(_stext - ORIGIN(rom));` to LONG(_stext);` to get it to work.

I squashed this change into the single update commit and added your SoB line. Let me know if you would rather me split out the patch

Reply:
You need to update past opentitan@1437bb418ef3b61edba0060a25c5c5add8902840.  Prior to that commit, the test_rom did not interpret the manifest header in compliance with the [manifest](https://docs.opentitan.org/sw/device/silicon_creator/rom_ext/docs/manifest/) definition.